ci/ray_ci/automation/docker_tags_lib.py

```diff
@@ -169,6 +176,88 @@ def delete_tag(tag: str, docker_hub_token: str) -> bool:
     return True
 
 
+def _is_release_tag(
+    tag: str,
+    release_versions: Optional[List[str]] = None,
+) -> bool:
+    """
+    Check if tag is a release tag & is in the list of release versions.
+    Args:
+        tag: Docker tag name
+        release_versions: List of release versions.
+            If None, don't filter by release version.
+    Returns:
+        True if tag is a release tag and is in the list of release versions.
+            False otherwise.
+    """
+    variables = tag.split(".")
+    if len(variables) != 3 and "post1" not in tag:
+        return False
```

Comment:
nit: give elements in the variables array some name for readability

(.., .., ..) = (variables[0], variables[1], variables[2])